lib/membrane_rtc_engine/engine.ex

```diff
@@ -22,58 +21,61 @@ defmodule Membrane.RTC.Engine do
   ```
 
   This will register your process to receive RTC Engine messages.
-  If your process implements `GenServer` behavior then all messages can be handled
-  by `c:GenServer.handle_info/2`, e.g.
-
-  ```elixir
-  @impl true
-  def handle_info(%Message.NewPeer{rtc_engine: rtc_engine, peer: peer}, state) do
-    Engine.accept_peer(rtc_engine, peer.id)
-    {:noreply, state}
-  end
-  ```
-
   You can register multiple processes to receive messages from an RTC Engine instance.
   In such a case each message will be sent to each registered process.
 
-  ## Client Libraries
-
-  RTC Engine allows creating Client Libraries that can send and receive media tracks from it.
-  The current version of RTC Engine ships with WebRTC Client Library which connects to the RTC Engine
-  via WebRTC standard.
-  Communication with Client Libraries is done using `Media Events`.
-  Media Events are control messages which notify about e.g. new peer joining to the RTC Engine.
-  When Client Library receives Media Event it can invoke some callbacks.
-  In the case of WebRTC Client Library, these are e.g. `onPeerJoined` or `onTrackAdded`.
-  When RTC Engine receives Media Event it can emit some messages e.g. `t:#{inspect(__MODULE__)}.Message.NewPeer.t/0`.
-  More about Media Events can be read in subsequent sections.
-  Below there is a figure showing the architecture of the RTC Engine working in conjunction with some Client Library.
-
-  ```txt
-      +--------------------------------- media events -----------------------------+
-      |                                (signaling layer)                           |
-      |                                                                            |
-      |                                                                            |
-  +--------+                 +---------+             +--------+               +---------+
-  | user   | <-   media   -> | Client  |             |  RTC   | <- media   -> | user    |
-  | client |      events     | Library | <- media -> | Engine |    events     | backend |
-  | logic  | <- callbacks -  |         |             |        | - messages -> | logic   |
-  +--------+                 +---------+             +--------+               +---------+
-  ```
-
   ## Peers
 
+  > ### Deprecation notice {: .warning }
+  > 
+  > **Peers are deprecated as of version 0.9.0**
```

Comment:
Probably 0.10.0 and we can write `... and are going to be removed in the future`